### DISCUSSION TITLE
Issue with deletion of account

### DISCUSSION THREAD

**Discussion by ruppo-912116** [community/community#23060] [New to GitHub]
<p>Hello everyone, I have a github account which i need to delete. I have the credentials but during logging in, it sends the verification code(6-digit) to my email account which was deleted 2 years ago. Now, i cannot delete that account. Can someone help?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If anyone can help, it’s GitHub Support: <a href="https://support.github.com/contact/account" class="inline-onebox">Accounts - GitHub Support</a></p>
<p>Us other users can’t modify your account, and I think you’d want to keep it that way. :wink:</p>

**Comment by tijko** [comment]
<p>I’m having an issue deleting an account that I do not have access to the email anymore too.</p>
<p>It was an email from a former employer which I do not have access too anymore.  Is there a way I can delete the account still?</p>
